### PR TITLE
add `service_available?` check by default

### PR DESCRIPTION
**Pull request by JeanMertz**
uses `HealthCheck#healthy?` which defaults to `true` by default

### CODE REVIEW

File: lib/eipiai/webmachine/resources/base.rb [hunk 1]
```diff
@@ -74,6 +74,13 @@ def query_keys
       []
     end
 
+    def service_available?
+      return true if Eipiai::HealthCheck.new.healthy?
+
+      response.headers['Retry-After'] = '60'
```
Comment (jurriaan) [inline lib/eipiai/webmachine/resources/base.rb]: Maybe set some caching headers too?

File: lib/eipiai/webmachine/resources/base.rb [hunk 2]
```diff
@@ -74,6 +74,13 @@ def query_keys
       []
     end
 
+    def service_available?
+      return true if Eipiai::HealthCheck.new.healthy?
```
Comment (jurriaan) [inline lib/eipiai/webmachine/resources/base.rb]: Maybe make HealthCheck a singleton if you really want it to a class?

Comment (JeanMertz) [inline lib/eipiai/webmachine/resources/base.rb]: https://github.com/blendle/eipiai/issues/19

File: lib/eipiai/webmachine/resources/base.rb (hunk 1 above)
Comment (JeanMertz) [inline lib/eipiai/webmachine/resources/base.rb]: https://github.com/blendle/eipiai/issues/20

**Pull request merged**